extensions/amp-story-subscriptions/0.1/test/test-amp-story-subscriptions.js

```diff
@@ -1,37 +1,75 @@
 import '../amp-story-subscriptions';
-import {htmlFor} from '#core/dom/static-template';
+import * as Preact from '#core/dom/jsx';
+
+import {afterRenderPromise} from '#testing/helpers';
+
+import {registerServiceBuilder} from '../../../../src/service-helpers';
+import {
+  Action,
+  AmpStoryStoreService,
+} from '../../../amp-story/1.0/amp-story-store-service';
+import {AmpStorySubscriptions} from '../amp-story-subscriptions';
 
 describes.realWin(
   'amp-story-subscriptions-v0.1',
   {
     amp: {
       runtimeOn: true,
-      extensions: ['amp-story-subscriptions:0.1'],
+      extensions: ['amp-story:1.0', 'amp-story-subscriptions:0.1'],
```

Comment:
I'm not sure if you need the `amp-story-subscriptions` extension here if you also create the element directly as `new AmpStorySubscriptions()`

Reply:
Interesting. Just tried that and it would fail complaining about not found function of 'subscriptionsEl.whenBuilt' and 'this.element.getAmpDoc'. Seems like it is still required to call some methods on the custom element.

Reply:
I see, we usually either call `buildCallback` directly or add the extension to this list, so if the tests work as intended, all good.

Reply:
Cool good to know!

Reply:
Yeah, that's mostly why we do it that way. In general to mock things it's easier to do it over the instanciated element than with the `.getImpl()`

Reply:
Cool SG. I was also confused by the usage of `.getImpl()`, which i cannot find the definition in `AMP.baseElement`. 

Calling it on an custom element would crash if you don't import the element's class manually, even though you don't use the class directly, e.g. `ampStoryEl.getImpl()` would complain `getImpl()` function not found if you don't import {AmpStory} from ../amp-story in the unit test file. What is the workaround here if I want to use `getImpl()`?

Reply:
`await whenUpgradedToCustomElement(el); const impl = await el.getImpl()`